doc/tools/docgram/orderedGrammar

```diff
@@ -894,7 +902,7 @@ command: [
 | "Typeclasses" "eauto" ":=" OPT "debug" OPT ( "(" eauto_search_strategy_name ")" ) OPT int
```

Comment:
Still have `int` here and in 2 other places.  You could change the underlying code or just do it with editing, e.g.

int: [
| integer
]

SPLICE: [
| int
]